testserver/settings.py

```diff
@@ -120,8 +120,11 @@
 NOTIFICATION_PREFERENCES_IS_VISIBLE = True
 
 NOTIFICATION_STORE_PROVIDER = {
-    "class": "edx_notifications.stores.sql.store_provider.SQLNotificationStoreProvider",
+    "class": "edx_notifications.stores.mongo.store_provider.MongoNotificationStoreProvider",
     "options": {
+        'host': 'localhost',
+        'port': 27017,
+        'database_name': 'notifications'
     }
```

Comment:
Hmmm. Let me think if there's a way we can have configuration definitions for different providers at the same time. I might make a commit here.